db-engine/src/main/java/com/splicemachine/db/iapi/types/DataTypeDescriptor.java

```diff
@@ -1756,6 +1756,10 @@ public void setChildren(DataTypeDescriptor[] children) {
      }
 
     public DataValueDescriptor getDefault() throws StandardException {
+        if (typeId.getTypeFormatId() == StoredFormatIds.BIT_TYPE_ID) {
+            return new SQLBit(new byte[typeDescriptor.getMaximumWidth()]);
```

Comment:
Why do we need to create it with maximumWidth instead of 0?

Reply:
We mimic what DB2 is doing. For char(n) for bit data, their default is a byte array of size n, filled with 0x0